item.py

```diff
@@ -0,0 +1,4 @@
+class Item:
+    def __init__(self, name, description, points):
+        self.name = name
+        self.description = description
```

Comment:
Missing some methods here! Could be easy things like examine, identify, things like that! Check out my suggestion!!